tests/Agent/IntegrationTests/UnboundedIntegrationTests/MySql/MySqlConnectorTests.cs

```diff
@@ -14,47 +15,55 @@
 
 namespace NewRelic.Agent.UnboundedIntegrationTests.MySql
 {
-    public abstract class MySqlConnectorTestBase : NewRelicIntegrationTest<RemoteServiceFixtures.MySqlConnectorBasicMvcFixture>
```

Comment:
We should remove RemoteServiceFixtures.MySqlConnectorBasicMvcFixture to prevent confusion